### DISCUSSION TITLE
DBeaver querying

### DISCUSSION THREAD

**Discussion by jeflsdean84** [dbeaver/dbeaver#10981] [Q&A] [answered]
I am needing some assistance when it comes to querying using Dbeaver. I have data and within the columns for the account numbers there are numbers with 10, 6, 8, and 9 digits. I only want the accounts that have 6 digits. How do I narrow down my query to only produce the accounts containing the 6 digit account numbers?

**Comment by p91paul** [comment, marked as answer]
Try "WHERE LENGTH (ACCOUNT_NAME) = 6"
